src/ScrumMaster/Command/SlackNotifierOutput.php

```diff
@@ -19,9 +20,22 @@ public function __construct(OutputInterface $output)
 
     public function write(SlackNotifierResult $result): void
     {
-        $keys = implode(', ', $result->ticketKeys());
-        $this->output->writeln("Total notifications: {$result->total()} ({$keys})");
+        $notificationTitles = $this->buildNotificationTitles($result->responseCodePerTickets());
+
+        $this->output->writeln("Total notifications: {$result->total()} ($notificationTitles)");
         $this->output->writeln("Total successful notifications sent: {$result->totalSuccessful()}");
         $this->output->writeln("Total failed notifications sent: {$result->totalFailed()}");
     }
+
+    private function buildNotificationTitles(array $stackTickets): string
```

Comment:
Because it's a private function, I would have pass the result object, instead of the array.